generators/server/templates/gradle/_profile_dev.gradle

```diff
@@ -31,7 +31,7 @@ task gulpConstantDev(type: GulpTask) {
     args = ["ngconstant:dev", "--no-notification"]
 }
     <%_ } else { _%>
-task webpack(type: <%= clientPackageManager.charAt(0).toUpperCase() + clientPackageManager.slice(1) %>Task, dependsOn: '<%= clientPackageManager %>_install') {
+task webpackBuildDev(type: <%= clientPackageManager.charAt(0).toUpperCase() + clientPackageManager.slice(1) %>Task) {
```

Comment:
I think before the webpack task also did a `yarn install` (via `dependsOn: 'yarn_install'`) it looks like it is missing now, but not sure if it is required.